webv2/helpers/helpers.go

```diff
@@ -37,16 +37,13 @@ const (
 	GOOGLE_SQL_DIALECT     = "Google Standard SQL"
 )
 
-const metadataDbName string = "spannermigrationtool_metadata"
-
-func GetMetadataDbName() string {
-	return metadataDbName
-}
-
 func GetSpannerUri(projectId string, instanceId string) string {
-	return fmt.Sprintf("projects/%s/instances/%s/databases/%s", projectId, instanceId, GetMetadataDbName())
+	return fmt.Sprintf("projects/%s/instances/%s/databases/%s", projectId, instanceId, constants.METADATA_DB)
 }
 
+// Creates the schema for the internal metadata database
+// JobDetails contains the execution data associated with a job, such as the Id of the aggregated monitoring dashboard
+// ShardExecutionData contains the execution data associated with a data shard, such as the Id of the dataflow job, datastream stream etc.
```

Comment:
These comments are stale. Please clean them up.

Nit: it would be good if we can add the comments closer to each table rather than on top.

Reply:
Done